src/ontology/config/properties.txt

```diff
@@ -19,6 +19,8 @@ http://www.w3.org/2004/02/skos/core#narrowMatch
 http://www.w3.org/2004/02/skos/core#relatedMatch
 http://www.w3.org/2004/02/skos/core#exactMatch
 http://www.w3.org/2004/02/skos/core#closeMatch
+rdfs:comment
 rdfs:label
 rdfs:seeAlso
 owl:deprecated
+dcterms:description
```

Comment:
## Updated `config/properties.txt`
There are some [properties we decided to map to](https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1483728680) that aren't included in this file, so I've added them.
[_Related: Added `icd11foundation-property-map.sssom.tsv`_](https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1510429828)

Based on my understanding, any properties not present in this file will be removed.

Reply:
If the command is redundant, remove to save CPU cycles. I think we should prefer using config/properties.txt to have a common solution for all ingests, rather than configuring this on an ingest by ingest basis.

Reply:
- [ ] @joeflack4 
Sounds good. Verify that when I convert the file with and without the `robot remove ... --annotation-properties` line, it is the same effect as just leaving the `properties.txt` line by itself. If that's the case, remove the `robot remove` one.

Reply:
@matentzn @twhetzel Indeed, effect was the same! Marking resolved.